cmake/kokkos_tribits.cmake

```diff
@@ -6,6 +6,12 @@ INCLUDE(GNUInstallDirs)
 
 MESSAGE(STATUS "The project name is: ${PROJECT_NAME}")
 
+IF(GTest_FOUND)
+  SET(KOKKOS_GTEST_LIB GTest::gtest)
+ELSE()  # fallback to internal gtest
+  SET(KOKKOS_GTEST_LIB kokkos_gtest)
+ENDIF()
```

Comment:
Do we still need these changes since we now always use us the internal gtest with Trilinos?

Reply:
We do

Reply:
Yes, because it's still possible to use either the internal copy or external GTest (without Trilinos).